src/network/mcpe/NetworkSession.php

```diff
@@ -1113,18 +1122,57 @@ public function syncAvailableCommands() : void{
 			//use filtered aliases for command name discovery - this allows /help to still be shown as /pocketmine:help
 			//on the client without conflicting with the client's built-in /help command
 			$lname = strtolower($firstNetworkAlias);
-			$aliasObj = new CommandEnum(ucfirst($firstNetworkAlias) . "Aliases", $aliases);
+			$aliasObj = count($aliases) > 1 ? new CommandEnum(ucfirst($firstNetworkAlias) . "Aliases", $aliases) : null;
+
+			$overloads = [];
+			foreach($command->getOverloads() as $overload){
+				//TODO: we should only send permissible overloads here
```

Comment:
Is there a reason this was left as todo? Does it cause problems with the client  or something?